website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Integrate with Apple Business Manager
+sidebar_label: Apple Business Manager
+support_level: authentik
+tags: [integration, apple, ssf, backchannel]
+authentik_version: "2025.2.0"
+authentik_enterprise: true
+authentik_preview: true
+---
+
+## What is Apple Business Manager?
+
+> Apple Business Manager is a web-based portal for IT administrators, managers, and procurement professionals to manage devices, and automate device enrollment.
```

Comment:
Can we explain the relationship between ABM, ASM, and ABE? Is this Integration guide for all three? Or does one only integrate ABM, and then the others don't need any configuration, but work off the ABM config?